libfs/fs.go

```diff
@@ -125,31 +127,40 @@ func (fs *FS) lookupOrCreateFile(
 	}
 }
 
-// OpenFile implements the billy.Filesystem interface for FS.
-func (fs *FS) OpenFile(filename string, flag int, perm os.FileMode) (
-	f billy.File, err error) {
-	fs.log.CDebugf(
-		fs.ctx, "OpenFile %s, flag=%d, perm=%o", filename, flag, perm)
-	defer func() {
-		fs.deferLog.CDebugf(fs.ctx, "OpenFile done: %+v", err)
-	}()
-
+func (fs *FS) lookupParent(filename string) (
+	parent libkbfs.Node, base string, err error) {
 	parts := strings.Split(filename, "/")
```

Comment:
Not sure if I've already suggested this for this instance, but `filepath.ToSlash(filename)` before splitting.